src/app/zap-templates/zcl/data-model/chip/microwave-oven-control-cluster.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <cluster>
+    <name>Microwave Oven Control</name>
+    <domain>Appliances</domain>
+    <description>Attributes and commands for configuring the microwave oven control, and reporting cooking stats.</description>
+    <code>0x050F</code>
```

Comment:
This happens to be the wrong cluster ID, though it's hard to see that from the spec for the cluster, where it's also wrong.  See https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/7779